javascript-source/handlers/raidHandler.js

```diff
@@ -35,6 +36,7 @@
         raidReward = $.getIniDbNumber('raidSettings', 'raidReward');
         raidOutMessage = $.getIniDbString('raidSettings', 'raidOutMessage');
         raidOutSpam = $.getIniDbNumber('raidSettings', 'raidOutSpam');
+        raidOutOfflineMessage = $.getIniDbNumber('raidSettings', 'raidOutOfflineMessage');
```

Comment:
Casting string to number

Reply:
Oh strange, that must have resurfaced from an older draft!  I remember that happening and fixing it the same day.

Reply:
Oh I see, the (nearly) same thing happens twice, and I did fix the first one but not the 2nd. :) Thanks! 

Do I need to somewhere accept your fix of my fix?

Reply:
Just push the fixing commit to the same branch that the PR was opened on and I will re-review it